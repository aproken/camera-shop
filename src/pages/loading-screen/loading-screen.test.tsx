import { render, RenderResult } from '@testing-library/react';
import LoadingScreen from './loading-screen';

describe('Component: LoadingScreen', () => {
  it('Тестирование компонента LoadingScreen', () => {
    const { container }: RenderResult = render(<LoadingScreen />);
    expect(container).toMatchSnapshot();
  });
});

