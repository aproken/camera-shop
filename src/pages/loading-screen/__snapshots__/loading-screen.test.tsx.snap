// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: LoadingScreen Тестирование компонента LoadingScreen 1`] = `
<div>
  <span
    class="loader"
    role="status"
  />
</div>
`;
